app/javascript/src/pages/home/bannerAction.js

```diff
@@ -34,50 +41,51 @@ const bindBanner = (bannerList) => {
   }
 
   const moveleft = (event) => {
-    // 向左,
-    // 移除外部最后一个, 加到内部最后一个
-    let newindex = currentindex + 2
-    if (newindex > bannerLength - 1) {
-      newindex = newindex - bannerLength
-    }
+    // left <-, current increase, 7 0 1 -> 0 1 2
     currentindex += 1
     if (currentindex > bannerLength - 1) {
       currentindex = currentindex - bannerLength
```

Comment:
It will be always like this, right?

```suggestion
      currentindex = 0
```